ansible/host_vars/linux-01.ih-eu-mda1.nimbus.sepolia.yml

```diff
@@ -29,7 +33,7 @@ origin_certs:
 
 nginx_sites:
   era_files:
-    - listen 80 default_server
```

Comment:
Why was this removed? Was it causing issues?

Reply:
yeah `ansible` would complain of port 80 being already configured. I don't have the exact error since this executed probably a week ago

Reply:
Fair enough.